src/Filtering/PossiblyUnusedClassesFilter.php

```diff
@@ -81,19 +96,39 @@ public function filter(
                 }
             }
 
+            // is excluded attributes?
+            foreach ($fileWithClass->getAttributes() as $attribute) {
+                foreach ($attributesToSkip as $attributeToSkip) {
+                    if ($this->shouldSkip($attribute, $attributeToSkip)) {
+                        continue 3;
+                    }
+                }
+            }
+
+            // is excluded attributes by method?
+            foreach ($fileWithClass->getAttributesByMethod() as $attributes) {
+                foreach ($attributes as $attribute) {
+                    foreach ($attributesToSkip as $attributeToSkip) {
+                        if ($this->shouldSkip($attribute, $attributeToSkip)) {
+                            continue 4;
+                        }
+                    }
+                }
+            }
```

Comment:
Seem a bit hard to read. Could you extract this to new method without using continue `x`?

Reply:
Refactored it.